src/User/Handler.hs

```diff
@@ -0,0 +1,21 @@
+{-# LANGUAGE OverloadedStrings #-}
+module User.Handler where
+
+import qualified Heist.Interpreted as I  
+import qualified Data.Text as T
+import           Data.Map.Syntax ((##))
+import           Snap.Snaplet.Heist
+import           Application
+import           User.Persistence
+import           User.Types
+
+handleUsers :: AppHandler
+handleUsers = do
+  users <- selectAllUsers
+  renderWithSplices "users" $ userSplices users
+  where
+    userSplices u = "users" ## I.mapSplices (I.runChildrenWith . userSplice) u
```

Comment:
I think we can separate a little bit the splices from the handler:
```
src/User/Splices.hs
```

Additionally, I think we should have a type for the information that is going to be **presented**:
```
data UserPresenter = 
    UserPresenter {
        upName :: String,
        upEmail :: String,
        upBirthday :: String,
    }
```

And we can have a function to transform it in a different module `src/User/Presenter.hs`:

```haskell
toUserPresenter :: User -> UserPresenter
toUserPresenter = ...
```

Then, we can add some unit tests for `toUserPresenter` to make sure we are going to display the information in a specific way.

@kinorius123 @sestrella @dbalseiro Please, let me know what you think. Perhaps, it is too much separation of concers for now.

Reply:
I agree... There should be a presenter. Good catch @CristhianMotoche ... maybe we should display the following:
- Name
- Email
- Age
- Income (with money format)

Then you can see the usefulness (is that even a word?) of a presenter

Reply:
@dbalseiro Great! Then @kinorius123 , please split that into different modules. Thanks!
@dbalseiro Maybe **benefit** is the word you're looking for?